rust_src/src/casefiddle.rs

```diff
@@ -1,18 +1,19 @@
 //! Case conversion functions.
-use lisp::defsubr;
 use lisp::LispObject;
+use lisp::defsubr;
 use remacs_macros::lisp_fn;
```

Comment:
remacs_macros belongs on its own line before the local modules. See other files in the directory for examples.